configs/records.yaml.default.in

```diff
@@ -0,0 +1,218 @@
+##############################################################################
+# *NOTE*: All options covered in this file should be documented in the docs:
+#
+#    https://docs.trafficserver.apache.org/records.yaml
+##############################################################################
+
+ts:
+  accept_threads: !!int '1'
```

Comment:
We don't need this, there is only 1 scenario where this would be needed and this is when you register configuration inside a plugin which core has no idea about the type at loading time(before the plugin gets to say `RecRegisterConfigXX`). I will add a description on this PR about this in the next hours.